packages/renderer/src/stitch-frames-to-video.ts

```diff
@@ -278,7 +280,7 @@ export const spawnFfmpeg = async (
 				audioCodecName,
 				// Set bitrate up to 320k, for aac it might effectively be lower
 				'-b:a',
-				'320k',
+				options.audioBitrate || '320k',
```

Comment:
Convention in Remotion codebase: Use `??` operator

Reply:
Maybe I could add this rule in the linting configuration in a dedicate PR ? 💡